rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-configuring.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-nodes-machinepools-configuring"]
+= Configuring machine pools
+:context: rosa-nodes-machinepools-configuring
+
+toc::[]
+This document describes how to configure machine pools with {product-title} (ROSA).
```

Comment:
```suggestion
toc::[]

This document describes how to configure machine pools with {product-title} (ROSA).
```

Also, self-referential.

Reply:
Yes, it is. It is also nearly directly copied language from another part of ROSA documentation, as shown here:
https://docs.openshift.com/rosa/rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html

This is used throughout ROSA documentation.

Reply:
I'm just calling what I see based on the guidelines that peer and merge reviewers use. If there is documentation about ROSA exceptions somewhere, I'm happy to look at it.

Reply:
I understand that. There are ROSA exceptions, although I'm not sure how well-documented they are (it is something we are working on). Given the fact that this is the pattern that is currently used in ROSA documentation, this is what I am following.

Reply:
Could also just be a mistake that is getting replicated. I would clarify this and fix it in the future if needed.